chol_analysis.py

```diff
@@ -1,10 +1,28 @@
-def HDL_analysis():
-    pass
-
-def LDL_analysis():
-    pass
-
-
+def HDL_analysis(HDL_level):
+    if HDL_level >= 60:
+        return 'Nromal'
```

Comment:
typo 'Nromal' should be 'Normal'

Reply:
By the way, this is not the error, this it is just me pointing out a different typo.

Reply:
I also just noticed that this is on my copy of the repository, not yours.  So, you can completely ignore this if you want.  I guess I didn't get enough sleep last night.